addons/udes_stock/models/stock_location.py

```diff
@@ -70,9 +70,7 @@ def _domain_height_category(self):
         help="Product category speed to match with location speed.",
     )
     u_size = fields.Float(
-        string="Location size",
-        help="""The size of the location in square feet.""",
-        digits=(16, 2),
+        string="Location size", help="""The size of the location in square feet.""", digits=(16, 2)
```

Comment:
Nit: Not strictly your change but can we remove the triple quotes in the help string?